markdown/developers/docs_guidelines.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Documentation
+subtitle: Guidelines for contributing to nf-core documentation.
+menu:
+  main:
+    weight: 10
+---
+
+# nf-core/website - Documentation contribution guidelines
+
+Many thanks for taking an interest in improving the nf-core documentation. To maintain consistency and clarity across our documentation, please follow these contribution guidelines.
+
+## General writing guidelines
+
+1. Use US English:
+
+Write in US English, not British English. For example, "color" instead of "colour" and "organize" instead of "organise."
+
```

Comment:
```suggestion
1. Use British English:

Write in British English, not US English. For example, "colour" instead of "color" and "organise" instead of "organize."

```